mu4/appshell/appshellmodule.cpp

```diff
@@ -40,6 +43,25 @@ std::string AppShellModule::moduleName() const
     return "appshell";
 }
 
+void AppShellModule::resolveImports()
+{
+    auto lr = framework::ioc()->resolve<framework::ILauncherUriRegister>(moduleName());
+    if (lr) {
+        lr->registerUri(Uri("musescore://home"),
+                        framework::ContainerMeta(framework::ContainerType::PrimaryPage));
+        lr->registerUri(Uri("musescore://notation"),
+                        framework::ContainerMeta(framework::ContainerType::PrimaryPage));
+        lr->registerUri(Uri("musescore://sequencer"),
+                        framework::ContainerMeta(framework::ContainerType::PrimaryPage));
+        lr->registerUri(Uri("musescore://publish"),
+                        framework::ContainerMeta(framework::ContainerType::PrimaryPage));
+        lr->registerUri(Uri("musescore://settings"),
+                        framework::ContainerMeta(framework::ContainerType::PrimaryPage));
+        lr->registerUri(Uri("musescore://devtools"),
+                        framework::ContainerMeta(framework::ContainerType::PrimaryPage));
```

Comment:
You can make `using framework`, then I think I fit in one line and it will be more readable

Reply:
fixed